src/Native/System.Native/pal_io.h

```diff
@@ -622,10 +622,11 @@ extern "C" void Sync();
  */
 extern "C" int32_t Write(int32_t fd, const void* buffer, int32_t bufferSize);
 
-
 /**
-* Gets the windows size of the terminal
-*
-* Returns 0 on success; otherwise, returns errorNo.
-*/
+ * Gets the windows size of the terminal
+ *
+ * Returns 0 on success; otherwise, returns errorNo.
+ */
 extern "C" int32_t GetWindowSize(WinSize* windowsSize);
+
+extern "C" int32_t ReadStdinUnbuffered(void* buffer, int32_t bufferSize)
```

Comment:
This doesn't appear to be the latest version of this file, which has IsATty here.  The function also needs a comment.